tableschema/types/string.py

```diff
@@ -17,6 +17,8 @@
 # Module API
 
 def cast_string(format, value, **options):
+    if isinstance(value, (int, float)):
```

Comment:
Not sure I agree with this.
One could argue that if we're stringifying ints and floats, we could stringify all other datatypes too.